### PR TITLE
Deploy heroku

### PR DESCRIPTION
**Pull request by SimonLab**
This PR add some configuration for running the application on Heroku, The branch can be rebased after #10 is merged

### CODE REVIEW

**Comment by nelsonic** [comment]
@SimonLab once #10 is merged we should merge `master` into this branch to see the `true` "diff". 👍

File: config/config.exs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+use Mix.Config
+
+# General application configuration
+config :dwylbot,
+  ecto_repos: [Dwylbot.Repo]
+
+# Configures the endpoint
+config :dwylbot, Dwylbot.Endpoint,
+  url: [host: "localhost"],
+  secret_key_base: System.get_env("SECRET_KEY_BASE"),
```
Comment (SimonLab) [inline config/config.exs:15]: Get the secret_key_base from Heroku environment variables

File: config/prod.exs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+#     config :phoenix, :serve_endpoints, true
+#
+# Alternatively, you can configure exactly which server to
+# start per endpoint:
+#
+#     config :dwylbot, Dwylbot.Endpoint, server: true
+#
+# Finally import the config/prod.secret.exs
+# which should be versioned separately.
+# import_config "prod.secret.exs"
```
Comment (SimonLab) [inline config/prod.exs:69]: We don't use prod.secrete.exs anymore and we get the configuration is defined directly in this file now

File: config/prod.exs [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  cache_static_manifest: "priv/static/manifest.json",
+  secret_key_base: System.get_env("SECRET_KEY_BASE")
+
+# Do not print debug messages in production
+config :logger, level: :info
+
+# Configure your database
+config :dwylbot, Dwylbot.Repo,
+  adapter: Ecto.Adapters.Postgres,
+  url: System.get_env("DATABASE_URL"),
```
Comment (SimonLab) [inline config/prod.exs:27]: Get the Postgres url from Heroku settings

File: config/prod.exs [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+# Finally, we also include the path to a manifest
+# containing the digested version of static files. This
+# manifest is generated by the mix phoenix.digest task
+# which you typically run after static files are built.
+config :dwylbot, Dwylbot.Endpoint,
+  http: [port: {:system, "PORT"}],
+  url: [scheme: "https", host: "dwylbot.herokuapp.com", port: 443],
+  force_ssl: [rewrite_on: [:x_forwarded_proto]],
+  cache_static_manifest: "priv/static/manifest.json",
+  secret_key_base: System.get_env("SECRET_KEY_BASE")
```
Comment (SimonLab) [inline config/prod.exs:19]: get the secrete key from Heroku

**Comment by SimonLab** [comment]
@nelsonic sorry I can't manage to reduce the diff. I might not have execute my rebase/merge the right way.  I added a few comments where I change the config to make it work with Heroku

**Review by nelsonic — APPROVED** [review]
@SimonLab looks great!
amaze

**Pull request merged**